app/src/main/java/org/oppia/app/player/state/StateViewModel.kt

```diff
@@ -47,22 +48,27 @@ class StateViewModel @Inject constructor() : ObservableViewModel() {
 
   fun getCanSubmitAnswer(): ObservableField<Boolean> = canSubmitAnswer
 
+  private fun getAnswerItemList(): List<StateItemViewModel> {
+    return if (isSplitView.get() == true) {
+      rightItemList
+    } else {
+      itemList
+    }
+  }
+
   fun getPendingAnswer(
-    statePlayerRecyclerViewAssembler: StatePlayerRecyclerViewAssembler
+    answerHandlerRetriever: (List<StateItemViewModel>) -> InteractionAnswerHandler?
```

Comment:
Optional, but could also call this ``retrieveAnswerHandler`` since it's a function.

Reply:
Done